src/lib/InstantSearch.ts

```diff
@@ -250,6 +250,8 @@ See ${createDocumentationLink({
       const routerOptions = typeof routing === 'boolean' ? undefined : routing;
       this.use(createRouterMiddleware(routerOptions));
     }
+
+    this.use(createTelemetryMiddleware());
```

Comment:
What do you think about this?: https://github.com/algolia/instantsearch-rfcs/pull/39#issuecomment-721798880

Reply:
We can add an option, but I'm not sure if it's needed

Reply:
For now we aren't adding an option